Source/JavaScriptCore/interpreter/CallFrame.h

```diff
@@ -219,6 +221,16 @@ using JSInstruction = BaseInstruction<JSOpcodeTraits>;
         CallSiteIndex callSiteIndex() const;
         CallSiteIndex unsafeCallSiteIndex() const;
 
+        // This flag can be only used from host functions.
+        bool ignoreResult() const
```

Comment:
exciting

Reply:
We first defer this thing to ensure that it is not perf regression. For now, I'll just add call_ignore_result, and then, leveraging this information in DFG or upper.